### PR TITLE
fix: rename `rollup` field

### PR DESCRIPTION
**Pull request by PJColombo**
### Checklist

- [ ] My change requires a documentation update, and I have done it.
- [x] I have added tests to cover my changes.
- [x] I have filled out the description and linked the related issues.

### Description
_**BREAKING CHANGE**_

It renames the `rollup` field to `category`.

#### Motivation and Context (Optional)
Not all blobs follow a rollup encoding standard (i.e [blobscriptions](https://blobscriptions.io)). This PR changes it to a more general and accurate label

### CODE REVIEW

File: packages/api/src/utils/schemas.ts [hunk 1]
```diff
@@ -27,16 +27,17 @@ const zodRollupEnums = [
   "zksync",
   "mode",
   "zora",
+  "unknown",
```
Comment (PabloCastellano) [inline packages/api/src/utils/schemas.ts:30]: "unknown" would make sense for rollups.

For categories I believe "other" or "uncategorized" are better choices.

**Review by PabloCastellano — COMMENTED** [review]
I left you a suggestion.

**Review by PabloCastellano — CHANGES_REQUESTED** [review]
Let's discuss a way for filtering blobs that don't belong to a rollup without introducing a breaking change for our API users

**Comment by PJColombo** [comment]
Closing this as it was decided to keep `rollup` enum and have a new `category` enum

**Pull request closed without merging**